superset/db_engine_specs/postgres.py

```diff
@@ -115,3 +119,28 @@ def convert_dttm(cls, target_type: str, dttm: datetime) -> Optional[str]:
             dttm_formatted = dttm.isoformat(sep=" ", timespec="microseconds")
             return f"""TO_TIMESTAMP('{dttm_formatted}', 'YYYY-MM-DD HH24:MI:SS.US')"""
         return None
+
+    @staticmethod
+    def get_extra_params(database: "Database") -> Dict[str, Any]:
+        """
+        For Postgres, the path to a SSL certificate is placed in `connect_args`.
+
+        :param database: database instance from which to extract extras
+        :raises CertificateException: If certificate is not valid/unparseable
+        :raises JSONDecodeError: If database extra json payload is unparseable
+        """
+        try:
+            extra = json.loads(database.extra or "{}")
+        except json.JSONDecodeError as ex:
+            logger.error(ex)
```

Comment:
will this error get surfaced to the user? probably better no to log it

Reply:
Good point, replaced with generic `SupersetException`.